### PR TITLE
part of the defaults

### PR DESCRIPTION
**Pull request by rshade**
This is some of what is missing and needs to be added and filled out. LICENSE, is a bigger problema s the name has spaces.

---
This change is [Reviewable](https://reviewable.io/reviews/sous-chefs/sc-generator/2)

### CODE REVIEW

**Comment by josephholsten** [comment]
Perhaps a short name map for the license? Like "apache" => the expanded license name and an appropriate template? And maybe an unknown license skips writing the LICENSE and merely sets it in metadata?

**Comment by rshade** [comment]
there is an apachev2.erb, that is correct, but you need a name to var mapping I guess.

File: templates/default/.travis.yml.erb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Use Travis's cointainer based infrastructure
+sudo: false
+addons:
+  apt:
+    sources:
+      - chef-stable-precise
```
Comment (shoekstra) [inline templates/default/.travis.yml.erb]: Let's change this to `chef-stable-trusty`?

Precise LTS goes EOL next month.

Comment (shortdudey123) [inline templates/default/.travis.yml.erb]: agree w/ `trusty`

Comment (josephholsten) [inline templates/default/.travis.yml.erb]: it's not enough to change the apt repo, see https://docs.travis-ci.com/user/trusty-ci-environment/

**Comment by rshade** [comment]
going to leave to @josephholsten to merge

File: templates/default/.travis.yml.erb [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Use Travis's cointainer based infrastructure
```
Comment (shoekstra) [inline templates/default/.travis.yml.erb]: it's a container, not a cointainer.

I'm unsure if for something trivial as this I should just push a new commit (cause I can?) or put a comment for someone to fix.

Comment (rshade) [inline templates/default/.travis.yml.erb]: sounds good to me, push a commit

**Pull request merged**